SPECS/llvm/llvm.spec

```diff
@@ -88,6 +89,9 @@ ninja check-all
 %{_includedir}/*
 
 %changelog
+* Wed Feb 09 2022 Chris Co <chrco@microsoft.com> - 12.0.1-3
+- Allow tools to link to libLLVM shared library
```

Comment:
Although, I was able to build the package, for what its worth, we have 3 tests that fail -
(I am digging for if we are missing anything)

```
msg="Failed Tests (3):"
msg="  LLVM :: DebugInfo/X86/vla-multi.ll"
msg="  LLVM :: tools/llvm-ar/error-opening-permission.test"
msg="  LLVM :: tools/llvm-elfabi/fail-file-write.test"


msg="Testing Time: 213.67s"
msg="  Unsupported      : 15625"
msg="  Passed           : 26433"
msg="  Expectedly Failed:    57"
msg="  Failed           :     3"
```

Reply:
Thanks for pointing this out Falak! I'll look into this closer and see how to resolve these 3 test failures.